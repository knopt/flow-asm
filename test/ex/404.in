5
3
-3.611828093961938
-5.036106894304595 -9.351008169549242 3.6958514377856098 0.6373352297310557 -0.3902640836905231 
2.3968864764508506 9.774541370832551 -5.329159439798725 -6.513519171897794 -1.077353892857607 
7.684826026303995 -3.27182908830612 1.238677758924755 -0.5998383808574488 -3.018397374898772 

1.5254480550340368 -8.333299182388858 -3.0953849138266527 
