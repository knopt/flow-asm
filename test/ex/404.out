-43.67311794183084 -136.44526288789004 61.46047026684347 36.96811035715796 20.496018876664444 
71.88466402556969 213.6620151604427 -109.10431442762493 -122.85217242348172 15.168339402692075 
123.5749380253925 -110.44105732912739 10.42168001896266 31.200450460127236 -6.1407529782997265 
